Update contributing.md (#8832)
PrairieLearn/PrairieLearn: docs/contributing.md

**Getting started as a collaborator:** Everyone should fork the main repository on GitHub. The full name of this repository is `PrairieLearn/PrairieLearn` (`"organization_name/repo_name"`); when you fork it, GitHub will create a repo named `user_name/PrairieLearn`. If you're not sure how to fork the repo, see [Github's instructions](https://help.github.com/articles/fork-a-repo/).

> Before you go further, you will need to set up SSH keys first (if you haven't already). If you're not sure how to set them up, see [Connecting to github with SSH](https://docs.github.com/en/authentication/connecting-to-github-with-ssh).

Reviewer: ```suggestion
Before you go further, you will need to set up SSH keys first (if you haven't already). If you're not sure how to set them up, see [Connecting to GitHub with SSH](https://docs.github.com/en/authentication/connecting-to-github-with-ssh).
```

> Before you go further, you will need to set up SSH keys first (if you haven't already). If you're not sure how to set them up, see [Connecting to github with SSH](https://docs.github.com/en/authentication/connecting-to-github-with-ssh).

Reviewer 2: ```suggestion
Before you go further, you will need to set up SSH keys first (if you haven't already). If you're not sure how to set them up, see [Connecting to GitHub with SSH](https://docs.github.com/en/authentication/connecting-to-github-with-ssh).
```

> Before you go further, you will need to set up SSH keys first (if you haven't already). If you're not sure how to set them up, see [Connecting to GitHub with SSH](https://docs.github.com/en/authentication/connecting-to-github-with-ssh).

Reviewer 3: ```suggestion
Before you go further, you will need to set up your own SSH keys and associate them with your GitHub account (if you haven't already). If you're not sure how to set them up, see [Connecting to GitHub with SSH](https://docs.github.com/en/authentication/connecting-to-github-with-ssh).
```
